vmaas_sync/vmaas_sync.py

```diff
@@ -115,6 +117,20 @@ def sync_cve_md(webhook_queue, page_size=5000):  # pylint: disable=too-many-bran
                 LOGGER.info('Finished importing CVE metadata (page: %s, page_size: %s, pages: %s)',
                             page, page_size, r_json['pages'])
                 if page >= r_json['pages']:
+                    to_delete = [(cve,) for cve in cves_in_db if cve not in all_rh_cves]
+                    if to_delete:
+                        LOGGER.info('Deleting unnecessary CVE metadata')
```

Comment:
I'd add here a count of CVEs that are going to delete. Just for information purposes.

Reply:
sure, done :)